Makefile.am

```diff
@@ -152,6 +152,35 @@ JSLINT_TARGET = jslint
 endif WITH_JSLINT
 lint: acilint apilint $(POLINT_TARGET) $(PYLINT_TARGET) $(JSLINT_TARGET)
 
+.PHONY: patchcheck
+patchcheck: all
```

Comment:
Python uses patchcheck, so I'm used to it. But I don't mind to switch to devcheck, develcheck, commitcheck or similar.

By the way, make has tab completion :)

> Use the tab, Luke!